debian/changelog

```diff
@@ -1,4 +1,4 @@
-cloud-init (22.2-64-g1fcd55d6-0ubuntu1~22.10.2) UNRELEASED; urgency=medium
+cloud-init (22.2-115-g6e498773-0ubuntu1~22.10.1) kinetic; urgency=medium
```

Comment:
made a minor change to version number in d/changelog, just force pushed  `22.2-115-g6e498773-0ubuntu1~22.10.1` instead of `22.2-115-g6e498773-0ubuntu1~22.10.1`

we don't need to `.2` because our official released version went from `22.2-64` to `22.2-115 in kinetic`